views/homepage.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Kainos Homepage</title>
+        <link rel="stylesheet" type="text/css" href="./css/style.css">
+        <link rel="icon" type="image/x-icon" href="https://companieslogo.com/img/orig/KNOS.L-8a0d29cc.png?t=1720244492">
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.7.1/jquery.min.js"></script>
+        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    </head>
+
+    <body>
+        <nav class="header">
+            <div class="headerformat">
+                <div class="middle">
+                  <div class="navbar-header">
+                    <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#menuoptions">
+                      <span class="icon-bar"></span>
+                      <span class="icon-bar"></span>
+                      <span class="icon-bar"></span>                        
+                    </button>
+                  </div>
+             
+                  <div class="collapse navbar-collapse" id="menuoptions">
+                    <ul class="nav navbar-nav">
+                    <div class="imagepositioning">
+                        <li>
+                            <img src="https://www.kainos.com/globalassets/images/5_logos/kainos_logo.png?mode=crop&width=400" alt="Logo" width="100" height="30">
+                          </li>
+                    </div>
+                     
+                    <div class="homepositioning">
+                        <li class="active"><a href="http://localhost:3000/homepage">Home</a></li>
+                    </div>
+             
+                    <div class="twopositioning">
+                        <li><a href="http://localhost:3000/jobRoles">Jobs</a></li>
+                    </div>
+             
+                    </ul>
+                  </div>
+                </div>
+             
+            </div>
+            </nav>
+
+
+            <img src="https://cdn.phenompeople.com/CareerConnectResources/KAINGB/images/End-User-Size-1-1643385498708.jpg" alt="homepageBackgroundImage" style="max-width: 100%; height: auto;" >
+
+        <!-- Footer Start -->
+                            <div class="footer-basic">
```

Comment:
move this section to a footer.html file and import it here so we can reuse on all pages